src/HJSON/HJSONParser.php

```diff
@@ -30,7 +30,7 @@ public function __construct()
     public function parse($source, $options = [])
     {
         $this->keepWsc = $options && isset($options['keepWsc']) && $options['keepWsc'];
-        $this->textArray = preg_split("//u", $source, null, PREG_SPLIT_NO_EMPTY);
```

Comment:
The third parameter should be integer type and `-1` by default.

The related reference is available [here](https://www.php.net/manual/en/function.preg-split.php).